server/svix-server/src/lib.rs

```diff
@@ -65,15 +71,15 @@ pub async fn run(cfg: Configuration, listener: Option<TcpListener>) {
     let with_api = cfg.api_enabled;
     let with_worker = cfg.worker_enabled;
 
-    let listen_address = SocketAddr::from_str(&cfg.listen_address).unwrap();
-
+    let listen_address =
+        SocketAddr::from_str(&cfg.listen_address).expect("Error coercing http address");
```

Comment:
I wouldn't call it coercing nor http address. Maybe "failed parsing server listen address".